services/ui-src/src/components/modals/AddEditReportModal.tsx

```diff
@@ -208,6 +238,7 @@ export const AddEditReportModal = ({
         formJson={form}
         formData={selectedReport?.fieldData}
         onSubmit={writeReport}
+        onChange={onChange}
```

Comment:
This makes sense and I agree with using onChange as the way to trigger the showing/hiding of the dropdown.